File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    max_x, max_y = 0, 0
    for x, y in sizes:
        if x > y:
            x, y = y, x
        max_x = max(max_x,x)
        max_y = max(max_y,y)
    return max_x*max_y

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    let maxX = 0;
    let maxY = 0;
    for(let [a, b] of sizes){
        if(a > b){
            let tmp = a;
            a = b;
            b = tmp;
        }
        maxX = Math.max(maxX, a);
        maxY = Math.max(maxY, b);
    }
    return maxX * maxY;
}